http.go

```diff
@@ -170,6 +174,13 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 			success = true
 		}
 
+		// If the response was gzipped and was transparently uncompressed
+		if resp.Uncompressed {
+			content_compressed = 1
+			if body, err := ioutil.ReadAll(resp.Body); err == nil {
+				content_length = int64(len(body))
```

Comment:
I doubt it is a bug, it looks pretty deliberate:

```go
resp.Body = &gzipReader{body: body}
resp.Header.Del("Content-Encoding")
resp.Header.Del("Content-Length")
resp.ContentLength = -1
resp.Uncompressed = true
```

I'm guessing it is as you say, they remove them so that noone tries to double-gunzip etc.